src/importexport/tabledit/internal/importtef.cpp

```diff
@@ -593,6 +594,79 @@ static void setInstrumentIDs(const std::vector<Part*>& parts)
     }
 }
 
+//---------------------------------------------------------
+//   fillGap
+//---------------------------------------------------------
+
+// Fill one gap (tstart - tend) in this track in this measure with rest(s).
+
+static void fillGap(Measure* measure, track_idx_t track, const Fraction& tstart, const Fraction& tend)
+{
+    Fraction ctick = tstart;
+    Fraction restLen = tend - tstart;
+    LOGN("measure %p track %zu tstart %d tend %d restLen %d len",
+         measure, track, tstart.ticks(), tend.ticks(), restLen.ticks());
+    while (restLen > Fraction(1, 256)) {
+        Fraction len = restLen;
+        TDuration d(DurationType::V_INVALID);
+        if (measure->ticks() == restLen) {
+            d.setType(DurationType::V_MEASURE);
+        } else {
+            d.setVal(len.ticks());
+        }
+        Segment* s = measure->getSegment(SegmentType::ChordRest, tstart);
+        addRest(s, track, d, len, muse::draw::Color::BLACK, false);
+        ctick   += len;
+        restLen -= len;
+    }
+}
+
+//---------------------------------------------------------
+//   fillGapsInFirstVoices
+//---------------------------------------------------------
+
+// Fill gaps in first voice of every staff in this measure for this part with rest(s).
+
+static void fillGapsInFirstVoices(Measure* measure, Part* part)
```

Comment:
It seems this method doesn't really depend on `part`, so maybe it's also possible to just iterate over all staves of the score in one go, instead of per part?